packages/zoe/src/zoeService/startInstance.js

```diff
@@ -61,6 +63,18 @@ export const makeStartInstance = (
 
     const instance = makeInstanceHandle();
 
+    const snoozedStrings = [];
+    const isSnoozed = string =>
+      snoozedStrings.some(s => string.indexOf(s) > -1);
+    const updateSnoozedList = strings => {
+      assert(
+        strings.every(s => typeof s === 'string' && isASCII.test(s)),
+        X`Snoozed strings (${q(strings)}) must be non-empty ascii strings.`,
+      );
+
+      snoozedStrings.splice(0, snoozedStrings.length, ...strings);
```

Comment:
Looks like this is to replace the contents of the array. I'm unfamiliar with this idiom. Is it common in the repo?

What's the advantage vs `let snoozedStrings` above and `snoozedStrings = strings` here? I guess mutability. But we could type the array as `Readonly<string[]>` to give confidence it won't be modified.

Reply:
> The advantage is immutability. const is enforced

Well sorta. The immutability of the binding is enforced, but it's an array and this line replaces the contents. So it's a full mutation. I'd say the `const` is more deceiving than helpful. It's a hoop to jump through to avoid rebinding the value.

I think it would be more direct to `let snoozedStrings`. The reader can tell at that point that it'll be reassigned. And it has the added benefit of making it possible to express that the array itself should not be mutated, with `Readonly<>`. Yes it can be overridden but at least the intent is expressed. And there's no loss to material runtime mutability.

Reply:
done.